controllers/assignments.py

```diff
@@ -693,6 +694,11 @@ def _get_qualified_questions(base_course, chapter_label, sub_chapter_label):
               (db.questions.practice == True)).select()
 
 
+# Gets invoked from lti to set timezone and then redirect to practice()
+def settz_then_practice():
+    return dict(course_name=request.vars.get('course_name', 'UMSI106'))
```

Comment:
You cannot hard code a course name (UMSI106) in code for general use.

Reply:
Professor @presnick has written this and he's not hard-coding it. He means if the 'course_name' does not exist, use 'UMSI106' as a default value. What else would you like to use as a default value?

Reply:
Well UMSI106 is only a Michigan course, so while it makes sense for a Michigan installation it really does not make sense for a general installation on runestone.academy

We really should have a setting in models/1.py  to provide a default value for this.